cob_bringup/components/raw3_base.launch

```diff
@@ -5,6 +5,7 @@
 
 	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
 	<arg name="sim" default="false"/>
+	<arg name="disable_stuck_detector" default="false"/>
```

Comment:
if stuck_detector needs to be disabled (in case @ipa-fmw's https://github.com/ipa320/cob_robots/pull/702/files#r131082757) does not help, I'd prefer positive rather than double-negative: 
`<arg name="enable_stuck_detector" default="true"/>`